src/test/java/tests/PriceAndNameOfProductTest.java

```diff
@@ -39,4 +44,22 @@ public void nameTest() {
         String nameInShoppingCart = shoppingCartPOMPage.nameInShoppingCart();
         Assert.assertEquals(nameInShoppingCart, nameOnPage, "Names are not equal");
     }
+
+    @Test
+    public void linkedinWithWaitTest() {
+        loginPOMPage = new LoginPOMPage(driver);
+        shoppingCartPOMPage = new ShoppingCartPOMPage(driver);
+        linkedinPage = new LinkedinPage(driver);
+        loginPOMPage.openRegistrationPage();
+        loginPOMPage.inputIntoEmail(WebUrl.EMAIL);
+        loginPOMPage.inputIntoPassword(WebUrl.PASSWORD);
+        loginPOMPage.loginButton();
+        removeTimeout();
+        linkedinPage.openLinkedin();
+        linkedinPage.newTabForLinkedin();
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.className("nav__logo-link")));
```

Comment:
Let's move wait into page. It could be a part of methods newTabForLinkedin, getLogo or could be a separate method. Test isn't a place to store locators, BTW we already have this locator in the page